### PR TITLE
spring-06 task done

### PR DESCRIPTION
**Pull request by less08**

### CODE REVIEW

File: src/main/java/com/dev/cinema/dao/MovieSessionDaoImpl.java [hunk 1]
```diff
@@ -102,4 +103,16 @@ public void delete(Long id) {
...
         }
     }
+
+    @Override
+    public Optional<MovieSession> getById(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            Query<MovieSession> query = session.createQuery("from MovieSession where id=:id",
+                    MovieSession.class)
+                    .setParameter("id", id);
+            return query.uniqueResultOptional();
```
Comment (Max-Pochepets) [inline src/main/java/com/dev/cinema/dao/MovieSessionDaoImpl.java:113]: you could use just `get(ClassType, id)`

File: src/main/java/com/dev/cinema/service/mapper/OrderMapper.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import com.dev.cinema.model.Order;
+import com.dev.cinema.model.Ticket;
+import com.dev.cinema.model.dto.OrderResponseDto;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderMapper {
+
```
Comment (Max-Pochepets) [inline src/main/java/com/dev/cinema/service/mapper/OrderMapper.java]:
```suggestion
```

**Review by Max-Pochepets — APPROVED** [review]

File: src/main/java/com/dev/cinema/controller/AuthenticationController.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+public class AuthenticationController {
+    private final AuthenticationService service;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/register")
+    public void registration(@RequestBody UserRequestDto requestDto) {
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/controller/AuthenticationController.java]:
```suggestion
    public void register(@RequestBody UserRequestDto requestDto) {
```
method names should be verbs

File: src/main/java/com/dev/cinema/service/mapper/OrderMapper.java (hunk 2 above)
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/service/mapper/OrderMapper.java]: redundant empty line

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**